cmd/neofs-lens/internal/objects/link.go

```diff
@@ -0,0 +1,39 @@
+package objects
+
+import (
+	"errors"
+	"os"
+
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	"github.com/spf13/cobra"
+)
+
+var linkCMD = &cobra.Command{
+	Use:   "link",
+	Short: "Inspect link object",
+	Run:   linkFunc,
+}
+
+func linkFunc(cmd *cobra.Command, _ []string) {
+	if vPath == "" {
+		common.ExitOnErr(cmd, errors.New("empty path to file"))
+	}
+
+	raw, err := os.ReadFile(vPath)
+	common.ExitOnErr(cmd, common.Errf("reading file: %w", err))
+
+	var link object.Link
+	err = link.Unmarshal(raw)
+	common.ExitOnErr(cmd, common.Errf("decoding link object: %w", err))
```

Comment:
i'd also support (as fallback) binary linking objects that can be downloaded via `object get --binary`. What u think?